webapp/src/pages/app/projects/[projectName]/repos/[repoName]/files/[revision]/[[...path]]/index.tsx

```diff
@@ -22,8 +23,12 @@ const FileContentPage = () => {
   }
   return (
     <Box p="2">
+      <Breadcrumbs path={router.asPath} omitIndexList={[0, 3, 5, 6]} suffixes={{ 4: '/list/head' }} />
```

Comment:
Not sure it is a good idea. 😅 
We may generalize `Breadcrumbs` to use in various situations. The number-based `omitIndexList` is unlikely to be used in complex cases.

Just random thoughts:
```tsx
// Create breakcrumbs for all path segments by default.
<Breadcrumbs path={router.asPath}/>

// Remove a segment or transform it into another.
<Breadcrumbs path={router.asPath}, 
  filter={(previousSegments, currentSegment, index) => {
    if (currentSegment === "ignorable") {
       return false;
    } 
    if (currentSegment === "replacable") {
        return "alternative";
    }
    ...
  }
}/>

<Breadcrumbs path={router.asPath}, 
             toLink={(segments) => segments.join("/") + "/list/head") />
```

I read the examples after I wrote. It does not look convenient to use. 😅
We may optimize the component later when we need such cases.

Reply:
Let's keep it as is. I don't think it is widely used in various cases.